core/config/docs/chains-evm.toml

```diff
@@ -424,3 +424,9 @@ GasLimit = 5400000 # Default
 FromAddress = '0x2a3e23c6f242F5345320814aC8a1b4E58707D292' # Example
 # ForwarderAddress is the keystone forwarder contract address on chain.
 ForwarderAddress = '0x2a3e23c6f242F5345320814aC8a1b4E58707D292' # Example
+# GasLimit controls the gas limit to be used for transactions sent via ChainWriter
```

Comment:
Let's rename the config section from `EVM.ChainWriter` to `EVM.Workflow` since the config is specific to the keystone workflows/keystone forwarder.

Reply:
Sounds good!

Reply:
@jmank88 What would break if we changed it? Assuming I changed all the references from ChainWriter to Workflow?

Reply:
I added this for the purposes of the original Keystone demo, the field isn't being used outside of the one staging cluster we have

Reply:
That's good, but I wonder if we should add a formal way to document experimental config so that we don't leave a confusing historical trail.